packages/ember/tests/reexports_test.js

```diff
@@ -34,6 +34,12 @@ moduleFor(
       }, /EXTEND_PROTOTYPES is deprecated/);
     }
 
+    ['@test Ember.NAME_KEY is deprecated']() {
+      expectDeprecation(() => {
+        Ember.NAME_KEY;
+      }, 'Using `Ember.NAME_KEY` is deprecated, override `.toString` instead');
```

Comment:
should make it clear that it is static toString.   MyClass.toString = () => 'MyClass'